ui/src/containers/InventoryLayout.js

```diff
@@ -95,10 +112,22 @@ const InventoryLayout = (props) => {
 
   const isSelected = (id) => selected.indexOf(id) !== -1
 
+  const date = new Date();
+
+  let day = date.getDate();
+  let month = date.getMonth() + 1;
+  let year = date.getFullYear();
+  
+  let currentDate = `${year}-${month}-${day}`;
```

Comment:
There are more concise ways to make a date in this format, please consider a different implementation